src/openfermion/ops/_fermion_operator.py

```diff
@@ -262,7 +262,7 @@ def __init__(self, term=None, coefficient=1.):
             with a scalar as calls an out-of-place multiplication.
 
         Args:
-            term (tuple of tuples, a string, or optional):
+            term (tuple or list of tuples, a string, or optional):
```

Comment:
Does this PR make sense without your other PR being merged? What do you want to do about that one?